plugins/exporter.koplugin/main.lua

```diff
@@ -38,7 +38,7 @@ local _ = require("gettext")
 
 -- migrate settings from old "evernote.koplugin" or from previous (monolithic) "exporter.koplugin"
 local function migrateSettings()
-    local formats = { "html", "joplin", "json", "readwise", "text", "my_clippings" }
+    local formats = { "html", "joplin", "json", "readwise", "text", "my_clippings", "memos" }
```

Comment:
It used to be sorted.

Could you please sort it again (both memos and my_clippings). Thanks!

Reply:
Is that the correct ordering: `'html', 'joplin', 'json', 'memos', 'my_clippings', 'readwise', 'text'` ?